packages/gestalt-cli/src/migrate.js

```diff
@@ -19,7 +20,8 @@ import {invariant} from 'gestalt-utils';
 import {get} from './cli';
 import 'babel-register';
 
-export default async function migrate() {
+export default async function migrate({directory = 'mutations', glob, url}: {directory: string, glob: string, url: string}) {
```

Comment:
can you break this line up? - I forgot to add something to 'contributing.md', but the rest of the code uses an 80 column ruler and splits using this format:

```
export default async function migrate({
  directory = 'mutations',
  glob,
  url,
}: {
  directory: string,
  glob: string,
  url: string
}): Promise {
```